docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -172,14 +168,14 @@ The `/login` endpoint authenticates mainframe user credentials and returns an au
   * Basic access authentication
   * JSON with user credentials
 
-When authentication is successful the response to the request is an empty body and a token is contained in a secure `HttpOnly` cookie named `apimlAuthenticationToken`. When authentication fails, a user gets a 401 status code.
+When authentication is successful, the response to the request is an empty body and a token is contained in a secure `HttpOnly` cookie named `apimlAuthenticationToken`. When authentication fails, a user gets a 401 status code.
```

Comment:
May be change "user" for "caller", because in fact the client application gets the authentication result and the user may present just any error page - up to client application implementation. Same is bellow in /query